boards/weact-f411ce/doc.txt

```diff
@@ -0,0 +1,66 @@
+/**
+@defgroup    boards_weact-f411ce WeAct-F411CE board
+@ingroup     boards
+@brief       Support for the WeAct-F411CE Board
+
+## Overview
+
+WeAct-F411CE is a board with the same form-factor as the blue/blackpill,
+but with an STM32F411CEU6 and a USB-C connector.
+
+It is available on sites like AliExpress for less than 4€.
+
+## Hardware
+
+![WeAct-F411CE](https://user-images.githubusercontent.com/1301112/69389644-eb5fb080-0ccc-11ea-8002-67d3db851250.png)
+
+### MCU
+| MCU        | STM32F411CE           |
+|:---------- |:--------------------- |
+| Family     | ARM Cortex-M4F        |
+| Vendor     | ST Microelectronics   |
+| RAM        | 128Kb                 |
+| Flash      | 512Kb                 |
+| Frequency  | up to 100MHz          |
+| FPU        | yes                   |
+| Timers | 8 (2x watchdog, 1 SysTick, 6x 16-bit)    |
+| ADCs       | 1x 12-bit             |
+| UARTs      | 3                     |
+| SPIs       | 5                     |
+| I2Cs       | 3                     |
+| RTC        | 1                     |
+| Vcc        | 2.0V - 3.6V           |
+| Datasheet  | [Datasheet](https://www.st.com/resource/en/datasheet/stm32f411ce.pdf) |
+| Reference Manual | [Reference Manual](https://www.st.com/content/ccc/resource/technical/document/reference_manual/9b/53/39/1c/f7/01/4a/79/DM00119316.pdf/files/DM00119316.pdf/jcr:content/translations/en.DM00119316.pdf) |
```

Comment:
Mind to align the left column of the table? (The links in the right column are too long to allow for consistent alignment there.)

Reply:
aligned.